.travis.yml

```diff
@@ -23,11 +23,7 @@ install:
 
 addons:
   apt:
-    sources:
-      - sourceline: 'deb https://dl.bintray.com/sbt/debian /'
-        key_url: 'https://bintray.com/user/downloadSubjectPublicKey?username=sbt'
     packages:
-      - sbt=0.13.16
```

Comment:
I think there is no effects apt package settings. Because we can use sbt in travis-ci without special settings.

https://docs.travis-ci.com/user/languages/scala/

>  Travis CI Scala builder will use sbt to build it.
> Thanks to paulp/sbt-extras the sbt version of your project is dynamically detected and used.
 
sbt/website travis job download `sbt-launch.jar` each time via sbt-extras script. does not use installed apt.

https://travis-ci.org/sbt/website/builds/263994877#L1712

```
 Downloading sbt launcher for 0.13.16:
  From  http://repo.typesafe.com/typesafe/ivy-releases/org.scala-sbt/sbt-launch/0.13.16/sbt-launch.jar
    To  /home/travis/.sbt/launchers/0.13.16/sbt-launch.jar
```